plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/analysers/DivisionByZeroAnalyser.java

```diff
@@ -57,7 +57,7 @@ static private boolean isZeroLiteral(Expression expr) {
 		if (expr instanceof NumericLiteral) {
 			NumericLiteral numLit = (NumericLiteral) expr;
 			BigDecimal litValue = numLit.getValue();
-			return litValue.equals(0);
+			return litValue.equals(new BigDecimal(0));
```

Comment:
You may want to use BigDecimal.ZERO instead of the constructor.

Reply:
done